src/foam/nanos/auth/PermissionedPropertyDAO.js

```diff
@@ -173,7 +173,6 @@ foam.CLASS({
 
       while ( e.hasNext() ) {
         PropertyInfo axiom = (PropertyInfo) e.next();
-        axiom.clear(obj);
```

Comment:
not sure why we removing this line, isnt it the point of this method to clear the properties that user doesnt have permissions to?